src/main/java/com/hermesworld/ais/galapagos/changes/impl/ChangeBase.java

```diff
@@ -104,11 +104,11 @@ public static ChangeBase publishTopicSchemaVersion(String topicName, SchemaMetad
         return new PublishTopicSchemaVersionChange(topicName, schemaVersion);
     }
 
-    public static ChangeBase TopicProducerAddChange(String topicName, List<String> topicProducerIds) {
-        return new TopicProducerAddChange(topicName, topicProducerIds);
+    public static ChangeBase TopicProducerAddChange(String topicName, String topicProducerId) {
```

Comment:
Better param name: `producerApplicationId` (would be consistent with other methods and types)

Also, **bad** method name. Should be `addTopicProducer`.